packages/repeater/src/api/HttpRepeatersManager.ts

```diff
@@ -0,0 +1,39 @@
+import { RepeatersManager } from './RepeatersManager';
+import {
+  CreateRepeaterRequest,
+  DeleteRepeaterRequest,
+  ListRepeatersRequest
+} from './commands';
+import { inject, injectable } from 'tsyringe';
+import { CommandDispatcher } from '@secbox/core';
+
+@injectable()
+export class HttpRepeatersManager implements RepeatersManager {
```

Comment:
This implementation is transport agnostic, it has no http dependencies.

```suggestion
export class DefaultRepeatersManager implements RepeatersManager {
```